protocol/src/main/protobuf/User.proto

```diff
@@ -27,6 +27,7 @@ enum RpcType {
   RESP_FUNCTION_LIST = 9;
 
   QUERY_RESULT = 10; // drillbit is reporting a query status change, most likely a terminal message, to the user
+  QUERY_WARNING = 12; // drillbit is sending a query warning to the user
```

Comment:
If we include this, we will break the ODBC client backward compatibility. It would be best to leave this out and include a warning object optionally in the QueryResult. Older clients will be able to handle this, they just won't return a warning to the application. Newer clients can be updated to return a warning to the application.